aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogNamespace.java

```diff
@@ -105,7 +105,7 @@ public void testNamespaceExists() {
   public void testListNamespace() {
     String namespace = createNamespace();
     List<Namespace> namespaceList = glueCatalog.listNamespaces();
-    Assert.assertTrue(namespaceList.size() > 0);
+    Assert.assertFalse(namespaceList.isEmpty());
```

Comment:
Nice, much easier to read 👍

Reply:
nit: we might want to switch all Junit4-style assertions to AssertJ in the AWS integration test module (in a separate PR)